winsup/cygwin/cygwin.din

```diff
@@ -1208,7 +1208,7 @@ putc_unlocked SIGFE
 putchar SIGFE
 putchar_unlocked SIGFE
 putenv SIGFE
-puts NOSIGFE
+puts SIGFE
```

Comment:
Is the removal of `_alloca = __alloca NOSIGFE` still needed with the change bellow?

Reply:
yes, the change is still needed. it removes the reference to not required implementation on aarch64. it looks like it is not needed on ix86 either.